src/main/java/mekanism/common/util/CableUtils.java

```diff
@@ -1,8 +1,6 @@
 package mekanism.common.util;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
I'm not a fan of it either. I've reinstalled my IDE recently and haven't configured it properly yet, so it does these things on its own 😛 . I'll change it if it's a problem.